frontend/src/api/GameTimer.ts

```diff
@@ -1,7 +1,7 @@
 export type GameTimer = {
   duration: number,
-  startTime: Date,
-  endTime: Date,
+  startTime: string,
+  endTime: string,
```

Comment:
Not sure - however, the reason why I changed this to string is because I think that more accurately reflects what is in the axios response (I think axios receives the date in string format but doesn't actually serialize it into a date object)

Reply:
Ah yeah, I think you're right -- thanks!